### DISCUSSION TITLE
Is it possible to use an .trivyignore like construct with `trivy aws`?

### DISCUSSION THREAD

**Discussion by hans-d** [aquasecurity/trivy#5422] [Q&A] [answered]
### Question

`trivy aws` (includng any options like filtering on services) produce all the findings. Can we add some `.trivyignore` construct (local file is file, eg combo of resource arn + id) to have some of the findings be excluded from the reporting?

### Target

AWS

### Scanner

Misconfiguration

### Output Format

None

### Mode

Standalone

### Operating System

linux

### Version

```bash
Version: 0.45.1
Policy Bundle:
  Digest: sha256:1df8ade71efc830877ca3b1130f83e0c6368e3a45b0d4c0f0418955501644054
  DownloadedAt: 2023-10-22 04:22:28.286157023 +0000 UTC
```

**Comment by hans-d** [comment]
looks like the ignore-policy provides a bit more complex (cause rego) way to do so

**Comment by hans-d** [comment, marked as answer]
seems that https://github.com/aquasecurity/trivy/pull/5359 is needed for the ignore-policy to work

**Reply by simar7** [reply]
You got it. It has been merged and will be part of the next release.
